packages/kolibri-tools/lib/i18n/ExtractStrings.js

```diff
@@ -50,11 +47,11 @@ ExtractStrings.prototype.apply = function(compiler) {
   var self = this;
   // Only do this in non-production mode, as otherwise the module detection code
   // does not work for .vue files.
-  if (process.env.NODE_ENV !== 'production') {
+  if (process.env.NODE_ENV === 'i18n-extract') {
```

Comment:
I think this is superfluous - only had this flag here previously because it was being invoked inside the webpack lifecycle, and in production mode we were unable to parse the AST of the Vue files properly, so we just got blank messages.

Reply:
I only added this (and the same kind of thing in the ProfileStrings.js file) so that we don't run the entire string extraction every time you start the dev server or `yarn build`.